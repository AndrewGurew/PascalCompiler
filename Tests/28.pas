type  Traffic_Light= (RED, YELLOW, GREEN);
var   Section: Traffic_Light;