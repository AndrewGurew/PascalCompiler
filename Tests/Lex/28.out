Position            Type                     Text                Value               
-------------------------------------------------------------------------------------
(1,1)               Type                     type                type                
(1,7)               Id                       Traffic_Light       Traffic_Light       
(1,20)              Equal                    =                   =                   
(1,22)              Left bracket             (                   (                   
(1,23)              Id                       RED                 RED                 
(1,26)              Comma                    ,                   ,                   
(1,28)              Id                       YELLOW              YELLOW              
(1,34)              Comma                    ,                   ,                   
(1,36)              Id                       GREEN               GREEN               
(1,41)              Right bracket            )                   )                   
(1,42)              Semicolon                ;                   ;                   
(2,2)               Var                      var                 var                 
(2,8)               Id                       Section             Section             
(2,15)              Colon                    :                   :                   
(2,17)              Id                       Traffic_Light       Traffic_Light       
(2,30)              Semicolon                ;                   ;                   
(2,30)              End of file              EndOfFile           EndOfFile           
